### DISCUSSION TITLE
API added, No response

### DISCUSSION THREAD

**Discussion by LucaChow** [lifan0127/ai-research-assistant#91] [Q&A]
I installed ARIA today, and tried to use it to summarize articles. I tried multiple times with different prompts, no response is generated: 
<img width="416" alt="Screenshot 2024-04-23 at 8 31 11 PM" src="https://github.com/lifan0127/ai-research-assistant/assets/56675913/a9f12f80-c39d-4f9e-bd2c-4b4ffb826b07">

The OpenAI API usage page suggests that there were activity and tokens generated:
<img width="325" alt="Screenshot 2024-04-23 at 8 34 32 PM" src="https://github.com/lifan0127/ai-research-assistant/assets/56675913/77c23622-a34e-42b7-9841-4b544aac5434">

But none was shown on the ARIA interface

**Comment by lifan0127** [comment]
@HerculesChow I suspect there are some errors. Could you please use the "thumb down" button to send you feedback? Then I will be able to see the chat history and troubleshoot further. Thanks!

**Comment by a-layden** [comment]
I am having the same difficulty. It just doesn't seem to work. Would be very grateful for some help. I have used the thumbs down button. Thank you!

**Reply by lifan0127** [reply]
Thanks for sharing the chat history. I see the model couldn't answer the question and then didn't return anything instead of saying "I don't know". This appears to be a new behavior and I will investigate further.

What is the specific item you used for the QA? Is it a journal article, web page, a PDF attachment etc?

**Comment by a-layden** [comment]
Thanks! It was item containing PDF. Journal article.

**Reply by lifan0127** [reply]
I tried a few times but couldn't reproduce it. Which model do you use, GPT-4 or GPT-4-Turbo? If possible, could you please share this article with me?

Have you encountered the same behavior for other questions?
